### PR TITLE
Added Active Storage, Action Text and Google Maps API to application.

### PR DESCRIPTION
**Pull request by ErikGmz**
# Description

What did you implemented/Why did you implemented this:

- Active Storage was installed and tested in the project.
- Action Text was installed and tested in the project.
- Google Maps API was installed and tested in the project.
- Scaffolded test model, controller and views (**_these will be deleted later_**).

Examples:

- Test registration with some images, enriched text and location data.

## Testing

- Run the Rails migrations, rails server command and check /test endpoints.

### CODE REVIEW

File: app/javascript/controllers/places_controller.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      const { Marker } = await google.maps.importLibrary("marker");
+      const position = {lat: parseFloat(this.latitudeTarget.value || this.latitudeTarget.innerHTML), 
+                        lng: parseFloat(this.longitudeTarget.value || this.longitudeTarget.innerHTML)};
+
+      /*
+      * Google Cloud Billing is currently deactivated.
+      *
+      * Load address autocompletion (this will be used only
+      * if Google Cloud Billing is activated).
+      */
```
Comment (MauricioNC) [inline app/javascript/controllers/places_controller.js:28]: Don't leave this kind of comments.

File: app/javascript/controllers/places_controller.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      });
+      
+      this.marker = new google.maps.Marker({
+        position: position,
+        map: this.map,
+        title: "Location",
+      });
+
+      // Add listeners in order to update marker's positions
+      // when either the latitude or longitude change.
```
Comment (MauricioNC) [inline app/javascript/controllers/places_controller.js:56]: Same here

File: app/javascript/controllers/places_controller.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            types: ['address'],
+            componentRestrictions: {'country': ['MX']},
+            fields: ['place_id', 'geometry', 'name']
+          }
+        );
+
+        autocomplete.addListener("place_changed", onPlaceChanged);
+      }
+
+      // Load Google Map.
```
Comment (MauricioNC) [inline app/javascript/controllers/places_controller.js:43]: Same here

File: app/javascript/controllers/places_controller.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+        this.map.setCenter(position);
+        this.map.setZoom(14);
+        this.marker.setPosition(position);
+      }
+
+      this.latitudeTarget.addEventListener('change', updateAddressOnMap);
+      this.longitudeTarget.addEventListener('change', updateAddressOnMap);
+
+      // This also will be used only if Google Cloud Billing is activated).
```
Comment (MauricioNC) [inline app/javascript/controllers/places_controller.js:69]: Same here

File: config/credentials.yml.enc [hunk 5]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+jz2OPIeRtpylG+Pg66NDMQM67QfkEkdvMRCOGIKl06gDl4bnaY8PnFk7HJjStox5eKkQtruzqegD9C1G7IaG7buqVOfWn2pzUnfxxQ/7S51e52sk/RlZZ3Vjl/ScpZU0gdF5Uf1BnWRjEo1i2vQk01cqvqgTaSMrZc/dPxXcb4HOw1Vv5v/7pR1loRq0uZ220B834+Ik7aFSZ+7b4wnuZiX6FdQmFeXE3gVM9WBF1gVtbsbv8nJgXyO/CIQEFrP3PDvmURQAwnY8F88dqSQgz09dyrcAeNMeDvpg2ArsjTTj82rvEvGFGc08QNtdYEoLd2klYRNbwL64o4fzvjDWQPgvCBg/q1RgsgpGo3HhfDpEgq6mH470iZsE6vBfOqtvMz/880pxLr2leJWfRknRNoIxtiiuap3qoaq2mWAYozIVf1qTh2c9mGId/QbfUh1u2PaX+q43ZNCvZXPMcOVM8z6+eHaTJkXSpA6+1DX2MMhn2fFGVQhCoP3Ey+nt7gnAKRmQnjnEiU09GDIuPgz5hckLBgqdJKBq7/4p70tFSt7lP3nLHWx4fMKeiQfJPQjH5eGqdCRCZTtS/gF6aq0t0O8QOktge/ytMBJzGeWud0j4Yt/00wLf+ldaxGS0SyTo8J2PieuabPYlliXQObLU1rbxGSY+gkrg/VJWdeCai16/5E8HlAnwLhMiWlNitAV1PdPmcVI6Kn68z1ZjQcAkDAc7Dj84kTtd6wYl+lNnqk27eCkUCRt4kRmVOFxOvHbRbNMvJJvnmGHnbuRkMdP5gwKC09wc1QqCMw3I3rvNz3NaCbhiLGLIo8wVn8AJwcX91Pb9hRbVPBqYIxeaslGR5/spGlBNze9t59z+KCAXKRenJ8F9EI60EMlNqZZjEnwKDfrOCu+oNEPC4BOxgG9aaaSRwMzrXs95I2rSTP25--NEKExB/Ik/Y23Cny--ZgoAa8TRPkAUkAXMdWb5Cg==
```
Comment (MauricioNC) [inline config/credentials.yml.enc:1]: This must be ignored!
Add this file to the .gitignore file

File: app/controllers/tests_controller.rb [hunk 6]
```diff
@@ -0,0 +1,76 @@
+class TestsController < ApplicationController
```
Comment (MauricioNC) [inline app/controllers/tests_controller.rb:1]: Why do you have a test controller?, Is this part of your project?

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/javascript/controllers/places_controller.js (hunk 1 above)
Comment (ErikGmz) [reply app/javascript/controllers/places_controller.js]: Fixed.

**Comment by MauricioNC** [comment]
You have many changes. Remember that you should do the minimum necessary changes for each PR yo do.

By this PR you are installing Action Text, connecting to the Google Maps API, installing Active Storage, etc. Just focus on one thing at a time.

File: app/javascript/controllers/places_controller.js (hunk 2 above)
Comment (ErikGmz) [reply app/javascript/controllers/places_controller.js]: Fixed.

File: config/credentials.yml.enc (hunk 5 above)
Comment (ErikGmz) [reply config/credentials.yml.enc:1]: Without that file none of the application's API will work. All of the API keys contained in that file are encrypted and the decryption key is already ignored.

Comment (MauricioNC) [reply config/credentials.yml.enc:1]: This file will be ignored and hidden in the master branch, but you already have it in the local repository.

File: app/controllers/tests_controller.rb (hunk 6 above)
Comment (ErikGmz) [reply app/controllers/tests_controller.rb:1]: It was created in order to test the Action Storage, Action Text and Google Maps API features. It's also planned to be removed (along with its model and views) in the future.

File: config/credentials.yml.enc (hunk 5 above)
Comment (MauricioNC) [reply config/credentials.yml.enc:1]: Also you will need this credential when you deploy your application. You are storing sensitive information so you must have to ignore it.

**Review by MauricioNC — APPROVED** [review]

File: config/credentials.yml.enc (hunk 5 above)
Comment (ErikGmz) [reply config/credentials.yml.enc:1]: Now I get what do you mean; I'm going to address that right now.

File: app/controllers/tests_controller.rb (hunk 6 above)
Comment (ErikGmz) [reply app/controllers/tests_controller.rb:1]: You're right. The reason I decided to create a specific scaffold for those features was that I didn't want to rush the planned project's model development.

**Review by juperameza — APPROVED** [review]

**Review by AnaCarrizales — APPROVED** [review]

**Pull request merged**